src/ape/__init__.py

```diff
@@ -1,8 +1,8 @@
 import signal
+import sys
 
-signal.signal(signal.SIGINT, lambda s, f: _sys.exit(130))
-
-import sys as _sys
+if "test" not in sys.argv and "pytest" not in sys.argv:
```

Comment:
by avoiding signaling when pytest is running, we can rely on the existing handlers